generated/python/test/trace.py

```diff
@@ -28,10 +28,19 @@
 trace = trace_pb2.Trace(project_id=PROJECT_ID, trace_id=id, spans=[span])
 traces = trace_pb2.Traces(traces=[trace])
 
+print 'testing patch'
 api.patch_traces(PROJECT_ID, traces)
+print 'patched'
 
+# wait for patch to propagate
```

Comment:
As I commented in #77, I'm even happier if you leave a link to a document to justify the sleep (and why 1 second is sufficient), if such document exists.